.github/actions/label-and-move-released-issues-v1/src/run.test.ts

```diff
@@ -26,7 +27,7 @@ const MOCK_LIST_LABELS = {
   description: "Something isn't working",
   color: 'f29513',
   default: true
-} as any
+} as unknown as Endpoints["GET /repos/{owner}/{repo}/labels"]["response"]
```

Comment:
Is this the recommended way of using the `Endpoints` type? This is gross 😅

Reply:
Yep 🤮  https://www.npmjs.com/package/@octokit/types#examples

Reply:
Wow... 🤮 indeed